src/platform_impl/macos/view.rs

```diff
@@ -273,9 +273,12 @@ declare_class!(
         fn set_marked_text(
             &self,
             string: &NSObject,
-            _selected_range: NSRange,
+            selected_range: NSRange,
             _replacement_range: NSRange,
         ) {
+            // TODO: We need some Nihonjin IME devs' help regarding how to use _replacement_range correctly.
+            // The "_replacement_range" is at least used in Japanese IMEs.
+            // Therefore, Nihonjin IME devs are more familiar with Japanese IMEs on mac.
```

Comment:
Nit: Feels overly verbose, maybe just:
```suggestion
            // TODO: Use `_replacement_range` correctly (known to be used by Japanese IMEs).
```